pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -110,6 +111,8 @@ type Alertmanager struct {
 	silencer *silence.Silencer
 	silences *silence.Silences
 
+	muteTimes map[string][]timeinterval.TimeInterval
```

Comment:
I think also a comment here to explain what `string` is here and how come each value is a slice of `timeinterval.TimeInterval`?

Reply:
Will do 👍